### DISCUSSION TITLE
Needs Account Switcher

### DISCUSSION THREAD

**Discussion by vmo-mgosling** [community/community#3943] [Mobile] [answered]
My work requires a separate GitHub account, the app should allow multiple accounts for these kinds of situations. Not a huge problem on Android because you can have different users or profiles but on iOS that's not an option.

**Comment by elisealix22** [comment, marked as answer]
Thank you so much for the request! I'll add this to our internal tracking.

**Reply by getaaron** [reply]
Any updates on when this will ship?

**Reply by eksrha** [reply]
I would also be very interested in a quick switch. Is there already something on the roadmap or an issue?
@elisealix22

**Reply by davidhackn** [reply]
Bunp, please add this asap!

**Comment by Wladefant** [comment]
Duplicate of #3337. Not good because that means the votes get divided up

**Comment by thanhcsf** [comment]
Hello, Is there any update on this issue?

**Comment by eliperkins** [comment]
Hey @vmo-mgosling and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.
